README.md

```diff
@@ -516,6 +516,7 @@ Table of Contents
   * [yopmail.fr](http://www.yopmail.fr/en/) — Disposable email addresses
   * [zoho.com](https://www.zoho.com/mail/) — Free Webmail and collaboration for up to 5 users
   * [SimpleLogin](https://simplelogin.io/) – Open source, self-hostable email alias/forwarding solution. Free 5 Aliases, unlimited bandwith, unlimited reply/send. Free for educational staffs (student, researcher, etc).
+  * [odmail](http://t.odmail.cn/) - Free, temporary email for OneDrive 5T.[OneDrive Signup Link](https://signup.microsoft.com/signup?sku=student)
```

Comment:
sorry we cant list things we cant read :)

Reply:
[I found a translation](https://onehack.us/t/giveaway-5tb-onedrive-office365-for-free/58747), but I think that this will not fit the theme of the repository.

Reply:
yeah, don't think so either, thank you